### ISSUE TITLE
CAM-GAN: Continual Adaptation Modules for Generative Adversarial Networks

### ISSUE THREAD

**Issue by chaerinkong** [reyllama/paper-reviews#174]
# TL; DR

- Train parameter-efficient group convolution layers for forget-less continual learning of GAN.

# Motivation

- Continual learning has received little attention in terms of generative tasks.
- Expansion-based methods have shown impressive performances.

# Formulation

- Three types of group convolutional layers are added.
- (1) 3x3 Group Conv
- (2) 1x1 Point Group Conv
- (3) 3x3 Residual Group Conv
- (1) and (2) can be made computation friendly through reparametrization introduced in RegVGG.

<img width="1161" alt="image" src="https://user-images.githubusercontent.com/46308407/165938665-4e8ee7d1-f5f4-4ff3-a74d-b2a992a15aa0.png">

- Task-similarity based Transfer Learning initializes task specific parameters with parameters of most relevant past task.
- The relevance is acquired through computing Fisher Information Matrix (FIM) with task-specific weights.

# Evaluation

<img width="1180" alt="image" src="https://user-images.githubusercontent.com/46308407/165939014-d59c87b9-3eb3-4bf7-b592-5115f39a7f7f.png">
<img width="1193" alt="image" src="https://user-images.githubusercontent.com/46308407/165939026-595c2ec4-79b0-466f-baf8-408eaeb942f8.png">
<img width="1167" alt="image" src="https://user-images.githubusercontent.com/46308407/165939042-86e4ec6a-8a5d-4cfa-92ce-89073c3e0827.png">
<img width="769" alt="image" src="https://user-images.githubusercontent.com/46308407/165939055-40b28e99-39d2-4de5-b6b1-38153388c3e7.png">
<img width="1172" alt="image" src="https://user-images.githubusercontent.com/46308407/165939066-9c62c496-6086-41d8-92e8-877e18f12adc.png">
<img width="1180" alt="image" src="https://user-images.githubusercontent.com/46308407/165939083-76951eb3-376d-4af8-aeff-96543c020225.png">
<img width="589" alt="image" src="https://user-images.githubusercontent.com/46308407/165939093-40e70373-df7c-45a5-ad22-3cfdb7da5b38.png">

# Inspiration

- Handling training instability is crucial: they added learnable residual connections to mitigate side effects of network depth.
